package.json

```
@@ -63,6 +63,9 @@
     "jest-get-type": "^22.4.3",
     "jest-matcher-utils": "^22.0.0"
   },
+  "engines": {
+    "node": ">=12"
```

```suggestion
    "node": "^12.13.0 || ^14.15.0 || >=16.0.0"
```

to avoid the EOL 13 and 15, plus only support LTS versions of 12 and 14

---

Why did you choose `12.13` instead of `12.0`? And `14.15` instead of `14.0`?

---

Also, do we want to be that prescriptive? We'd add Node 17 to that list when it comes out? Then remove it when Node 18 comes out?

---

Stylistically, it looks like `^` or `>=` can both be used. Do we have a preference?

---

> Why did you choose `12.13` instead of `12.0`? And `14.15` instead of `14.0`?

Those were the first LTS releases of those majors